packages/itwin/one-click-lca-widget/package.json

```diff
@@ -73,17 +84,32 @@
     "nyc": "^15.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.2",
+    "redux": "^4.1.0", 
     "rimraf": "^3.0.2",
     "ts-jest": "^27.1.3",
     "typescript": "~4.3.0"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.2",
```

Comment:
are all of these actually peer deps? or did you just add whatever dev deps to here?
transitive peers that are missing, which are reported by the `rush update` should be added as dev deps.
you guys should review these peer deps, if theyre only being used to import types then they can remain as dev deps. and any pure/effect-free usage of classes can also probably be a dev dep

Reply:
Yeah, my bad. I added all the dependencies reported by ```rush update``` to peer dependencies and dev dependencies. Guillermo took a look and decided that the original peer dependencies were good so I removed what I added.